pkgs/development/python-modules/hsh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, commandlines
+, unittestCheckHook
+, pexpect
+, naked
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "hsh";
+
+  version = "1.1.0";
```

Comment:
```suggestion
  version = "1.1.0";
  pyproject = true;
```

Reply:
This is already added on the line below

Reply:
I'll mark it as resolved now that it is fixed.